app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
+a.Bar   {
```

Comment:
Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space